swarmrl/engine/resobee.py

```diff
@@ -0,0 +1,156 @@
+"""
```

Comment:
Can you double check that we have doc strings and type hints in all methods here.

Reply:
Added doc strings and type hints everywhere.